griptape/drivers/memory/conversation/base_conversation_memory_driver.py

```diff
@@ -1,17 +1,17 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING
 
 from griptape.mixins import SerializableMixin
 
 if TYPE_CHECKING:
-    from griptape.memory.structure import BaseConversationMemory
+    from griptape.memory.structure import Run
 
 
 class BaseConversationMemoryDriver(SerializableMixin, ABC):
     @abstractmethod
-    def store(self, memory: BaseConversationMemory) -> None: ...
+    def store(self, runs: list[Run], metadata: dict, *, overwrite: bool = False) -> None: ...
```

Comment:
i think it would be used per call to `store`. appending is default, if i pass overwrite (previous default behavior), it gets cleared out

Reply:
Alternative idea: how about adding a `clear()` (or `reset()`) method instead of `overwrite`?

(Or am I missing understanding the behavior?)

Reply:
its basically just a way to either "append" to the runs that are already stored vs replacing them. the existing behavior today always overwrites them. maybe a `clear` method on the driver makes sense. it would make it more explicit at least, i dont forsee the framework really needing to hook into that method for normal operation.